includes/base/widget-base.php

```diff
@@ -480,7 +480,13 @@ protected function add_render_attributes() {
 
 		$settings = $this->get_settings();
 
-		$this->add_render_attribute( '_wrapper', 'data-widget_type', $this->get_name() . '.' . ( ! empty( $settings['_skin'] ) ? $settings['_skin'] : 'default' ) );
+		$this->add_render_attribute(
+			'_wrapper',
+			[
+				'data-e-bg-lazyload-selector' => '.elementor-widget-container',
+				'data-widget_type' => $this->get_name() . '.' . ( ! empty( $settings['_skin'] ) ? $settings['_skin'] : 'default' ),
+			]
+		);
```

Comment:
why do we still need it?
and why a widget needs to know about the implementation of some other module?

Reply:
Not relevant, the file content is updated.